app/controllers/hearings/hearing_day/filled_hearing_slots_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Hearings::HearingDay::FilledHearingSlotsController < ApplicationController
+  def index
+    hearing_day = HearingDay.find_by(id: params[:hearing_day_id])
```

Comment:
```suggestion
    hearing_day = ::HearingDay.find_by(id: params[:hearing_day_id])
```
Force it to use the top level "HearingDay". This fixes the rspec error that's showing up because when it gets an invalid hearing_id it tries to access "HearingDay" like this
`uninitialized constant Hearings::HearingDay::FilledHearingSlotsController::HearingDay`

Reply:
thanks! i added the change